x-pack/plugin/otel-data/src/main/resources/index-templates/metrics-otel@template.yaml

```diff
@@ -27,14 +27,3 @@ template:
       data_stream.type:
         type: constant_keyword
         value: metrics
-    dynamic_templates:
-      - ecs_ip:
-          mapping:
-            type: ip
-          path_match: [ "ip", "*.ip", "*_ip" ]
-          match_mapping_type: string
-      - all_strings_to_keywords:
-          mapping:
-            ignore_above: 1024
-            type: keyword
-          match_mapping_type: string
```

Comment:
Why are these removed?

Reply:
This is a leftover from https://github.com/elastic/elasticsearch/pull/114042. The content has moved to [`ecs-tsdb@mappings`](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/otel-data/src/main/resources/component-templates/ecs-tsdb%40mappings.yaml).